### PR TITLE
Fixes gamebreaking oversight introduced in #8541

### PR DESCRIPTION
**Pull request by Rukofamicom**
## About The Pull Request

* Changes plasma ignition temperature to 320
Humans are hot enough to detonate plasma immediately upon injection, resulting in extraordinarily bad interactions

* Adds a check to plasma ignition proc to see if it is a reagent reaction
This proc was written with the intention of things made *out of* plasma igniting. It should not qdel things that are not literally *made of* plasma, it should only be the plasma burning up in a brilliant flash

## Why It's Good For The Game

The above two issues combined to create a perfect storm - injecting players with ordinary plasma immediately qdels them. This is an absolutely gamebreaking bug. 

This additionally prevents machines such as chemmasters and sleepers from being hit with qdel during a plasma flash

## Testing Photographs and Procedure

dreamseeker_L6FSFoDU1x

dreamseeker_xmtVPkyMSi

## Changelog
:cl:
fix: Plasma ignition no longer calls qdel on reagent reactions, and its reaction temperature was increased to account for normal human body temperatures.
/:cl:

### CODE REVIEW

**Review by ike709 — APPROVED** [review]

**Comment by ike709** [comment]
Not ideal but I'm willing to cope if another maintainer agrees.

File: code/game/atoms.dm [hunk 1]
```diff
@@ -1627,7 +1627,9 @@
 		//Regardless of power, whatever is burning will go up in a brilliant flash with at least a fizzle
 		playsound(T,'sound/magic/fireball.ogg', max(strength*20, 20), 1)
 		T.visible_message("<b><span class='userdanger'>[src] ignites in a brilliant flash!</span></b>") 
-		if(isturf(src))
+		if(reagent_reaction)
```
Comment (ike709) [inline code/game/atoms.dm]:
```suggestion
		if(reagent_reaction) // Don't qdel(src). It's a reaction inside of something (or someone) important.
```

**Review by ike709 — APPROVED** [review]
Better fix is better.

**Comment by itsmeow** [comment]
This is hilarious I can't believe I didn't see this earlier

**Pull request merged**